Port-pots web page

msdreams opened the pull request:
[DEMO LINK](https://msdreams.github.io/Potr_Pots/)

IvanFesenko reviewed (changes requested):
Greate gob.
- remove autocomplete styles
![2024-01-31_15h45_03](https://github.com/mate-academy/Potr_Pots/assets/61904995/27a3cf5d-5592-4946-99b5-da43aa25816a)
- Do not use input type="number" for credit card
![2024-01-31_15h45_33](https://github.com/mate-academy/Potr_Pots/assets/61904995/59980f20-510a-4272-b18f-2be64ed69762)
- Same about styles also submit should clear form - now it's not working
![2024-01-31_15h47_56](https://github.com/mate-academy/Potr_Pots/assets/61904995/b566aca9-ca33-47c7-b58a-600db81cf48e)
- it should be links
![2024-01-31_15h48_37](https://github.com/mate-academy/Potr_Pots/assets/61904995/6c6a3571-a935-47ed-84a4-d8e4de212498)
- disable scroll in case menu is open, also content overlapping
![2024-01-31_15h49_23](https://github.com/mate-academy/Potr_Pots/assets/61904995/3fb6945c-86fe-4e04-bc27-8834cf1d6fe7)
- Horizontal scroll on mobile resolution

msdreams commented:
can't disable scroll in case menu is open, can you help me? I added changes in the target condition for menu (page.css file) and it doesn't work :

  &__menu {
    position: fixed;
    top: 0;
    right: 0;
    transform: translateX(100%);
    transition: transform 0.3s;

    &:target {
      transform: translateX(0);

      .page__body {
        overflow-y: hidden;
      }
    }
  }

Viktor-Kost reviewed (changes requested):
### Nice!

### To improve:
1. fix icons. should change from `plus` to `minus`
https://github.com/mate-academy/Potr_Pots/assets/117587463/dfec02c9-95ce-438a-92fb-13aec305cb87

2. remove autocomplete styles as previous mentor asked and fix submit button styles
<img width="452" alt="Screenshot 2024-02-01 at 23 22 12" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/c0ae913f-2c38-4b83-8b02-eb144f929c98">

3. make all form fields required
4. it's a placeholder not an input value
<img width="1901" alt="Screenshot 2024-02-01 at 23 24 32" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/b41fa871-36cd-4298-8dd4-c5f4c4401c90">

5. use svg files for icons all over the page.
<img width="335" alt="Screenshot 2024-02-01 at 23 25 40" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/0b1f1bd3-61ff-4cd0-848d-8e5f8d528484">

6. add hover effect to all links
7. this is a link. should invoke a call
<img width="232" alt="Screenshot 2024-02-01 at 23 26 40" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/26ba48ac-e0b4-474a-82f0-08713ac2b957">

8. remove body scroll when modal is opened. you can make you `body` to be fixed
9. fix markup
<img width="420" alt="Screenshot 2024-02-01 at 23 29 08" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/ff7c67d3-7b06-424d-a529-a10debc3171f">

<img width="498" alt="Screenshot 2024-02-01 at 23 29 51" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/5dc3aed5-37bf-43b9-bf79-8c922307cb51">

<img width="489" alt="Screenshot 2024-02-01 at 23 31 02" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/466ac13d-8a27-4216-945b-d0c873877007">

<img width="475" alt="Screenshot 2024-02-01 at 23 32 42" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/e46178ea-fdb1-4001-a2cd-7ffbd1c93a61">

<img width="440" alt="Screenshot 2024-02-01 at 23 33 50" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/b1c5e8fe-083c-4532-b880-fdd563145fa3">

<img width="495" alt="Screenshot 2024-02-01 at 23 34 58" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/b2a7860c-f62b-492d-ac8b-c2eb0b8b61c9">

<img width="250" alt="Screenshot 2024-02-01 at 23 36 37" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/7450ac50-ebf3-4483-b261-ca93995afb49">

<img width="207" alt="Screenshot 2024-02-01 at 23 37 16" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/33cf5d2e-5fdb-4b37-9312-333357953a11">

<img width="394" alt="Screenshot 2024-02-01 at 23 38 38" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/82402ec5-8227-4426-ba06-ec579d5d015c">

<img width="678" alt="Screenshot 2024-02-01 at 23 39 25" src="https://github.com/mate-academy/Potr_Pots/assets/117587463/0ac469bc-e91e-4b3c-9012-a3d287b8f7ff">

anastasiiavorobiova reviewed (changes requested):
Great work! 
<img width="359" alt="Screenshot 2024-02-20 at 17 07 04" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/9b12b5d4-7f87-42e7-ae6a-5de2e6876ae7">
<img width="499" alt="Screenshot 2024-02-20 at 17 07 20" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/2f49f811-0e04-4601-a0b1-b15f95e0bee8">
Check the layout here 
<img width="584" alt="Screenshot 2024-02-20 at 17 07 28" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/8f4ba733-e02a-43cd-be93-436d35e4c334">
<img width="556" alt="Screenshot 2024-02-20 at 17 04 29" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/41f2cdd8-758f-4bca-98e9-5544cb536c00">
Consider fixing styles for the button. A user shouldn't be able to resize the text area here. Also, the form fields should be required. The page shouldn't be reloaded after submission, just cleared.

Esceype reviewed (changes requested):
Please add validation to the email field, just change the type of it.
And submit button still not fixed. You just need to add eventListener or change type of button to `submit` and add attribute on form tag:
`onsubmit="this.reset(); return false;"`

IvanFesenko reviewed (approved):
Approved
